core/src/main/java/com/linecorp/armeria/common/MediaType.java

```diff
@@ -479,6 +480,10 @@ private static MediaType addKnownType(MediaType mediaType) {
     public static final MediaType XRD_UTF_8 = createConstantUtf8(APPLICATION_TYPE, "xrd+xml");
     public static final MediaType ZIP = createConstant(APPLICATION_TYPE, "zip");
 
+    public static Set<MediaType> knownTypes() {
+        return KNOWN_TYPES.keySet();
```

Comment:
and I'm not sure if it's a good idea to add this to the public API. What's the definition of 'known' here? It seems confusing to me.